### PR TITLE
#2256 base theme cat links are not seo friendly

### PR DESCRIPTION
**Pull request by diashalabi**

### CODE REVIEW

File: addons/default/visiosoft/base-theme/resources/views/partials/categories-web.twig [hunk 1]
```diff
@@ -5,26 +5,22 @@
             <img src="{{ url('files/' ~ main_category.icon.path) }}" alt="{{ main_category.name }}"
                  class="img-responsive">
         {% else %}
-            <img src="{{ img('visiosoft.theme.base::images/default-categories-icon.png').url }}"
-                 alt="{{ main_category.name }}"
-                 class="img-responsive">
+            <img src="{{ img('visiosoft.theme.base::images/default-categories-icon.png').url }}" class="img-responsive"
```
Comment (spektra2147) [inline addons/default/visiosoft/base-theme/resources/views/partials/categories-web.twig:8]: please set category_icon variable

File: addons/default/visiosoft/base-theme/resources/views/partials/categories-web.twig [hunk 2]
```diff
@@ -5,26 +5,22 @@
...
         {% endif %}
-        <a href="{{ url_route('visiosoft.module.advs::list') }}?cat={{ main_category.id }}"
-           class="main-category">
+        <a href="{{ url_route('adv_list_seo', [main_category.slug]) }}" class="main-category">
             <b>{{ main_category.name }}</b>
         </a>
-        <ul class="p-0" style="">
+        <ul class="p-0">
             {% set sub_categories = entries('cats','category').where('parent_category_id', main_category.id).where('deleted_at', null).orderBy('sort_order').get() %}
             {% for sub_category in sub_categories|slice(0,sub_categories_limit) %}
```
Comment (spektra2147) [inline addons/default/visiosoft/base-theme/resources/views/partials/categories-web.twig:16]: please remove loop.Create each

Comment (diashalabi) [reply addons/default/visiosoft/base-theme/resources/views/partials/categories-web.twig:16]: what

File: addons/default/visiosoft/base-theme/resources/js/script.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+$('.show-all').on('click', function () {
```
Comment (spektra2147) [inline addons/default/visiosoft/base-theme/resources/js/script.js:1]: which show_all? for categories or for ads.

please change class name

File: addons/default/visiosoft/base-theme/resources/views/partials/categories-mobile.twig [hunk 4]
```diff
@@ -2,14 +2,13 @@
...
-        <a href="{{ url_route('visiosoft.module.advs::list') }}?cat={{ main_category.id }}"
-           class="main-category">
+        <a href="{{ url_route('adv_list_seo', [main_category.slug]) }}" class="main-category">
             {% if main_category.icon.path != "" %}
                 <img src="{{ url('files/' ~ main_category.icon.path) }}" alt="{{ main_category.name }}"
                      class="img-responsive">
             {% else %}
-                <img src="{{ img('visiosoft.theme.base::images/default-categories-icon.png').url }}" alt="{{ main_category.name }}"
-                     class="img-responsive">
+                <img src="{{ img('visiosoft.theme.base::images/default-categories-icon.png').url }}"
```
Comment (spektra2147) [inline addons/default/visiosoft/base-theme/resources/views/partials/categories-mobile.twig:10]: create icon variable

**Review by spektra2147 — APPROVED** [review]

**Review by fatihalp — APPROVED** [review]

**Pull request merged**